charts/misc-apps/templates/downscaler.yaml

```diff
@@ -0,0 +1,33 @@
+{{ if .Values.downscaler.enabled }}
+{{ template "argoconfig.application" (list . "misc-apps.downscaler") }}
+{{ end }}
+
+{{- define "misc-apps.downscaler" -}}{{- $app := unset .Values.downscaler "enabled" -}}{{- $name := default $app.namespace $app.name -}}
```

Comment:
If I understand this logic corectly, the last part of this line (in `$app.namespace`) does actually reference `downscaler.namespace`, that was recently removed in favour of `downscaler.destination.namespace`.
However, I am also not sure if we actually need a default value for `name`, since a default for `$app.name` is given in values.yaml. (Assuming I correctly understood this syntax.)
So I would either omit providing a default for `name` based on the namespece completely here, or otherwise I think one should use ` $app.deployment.namespace`.

Reply:
it uses `destination.namespace` as default again: 3ece9eb1ccf527e93e7ded1ab7e5ca00e2880f3f

I also added task to #84 so we don't forget to document how it works